deploy/evmos/121_deploy_EvmosBTCPool.ts

```diff
@@ -0,0 +1,58 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { execute, get, getOrNull, log, read, save } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // Manually check if the pool is already deployed
+  const SaddleEvmosBTC = await getOrNull("SaddleEvmosBTC")
+  if (SaddleEvmosBTC) {
+    log(`reusing "EvmoswBTCTokens" at ${SaddleEvmosBTC.address}`)
+  } else {
+    // Constructor arguments
+    const TOKEN_ADDRESSES = [
+      (await get("WBTC")).address,
+      (await get("RENBTC")).address,
+    ]
+    const TOKEN_DECIMALS = [8, 8]
+    const LP_TOKEN_NAME = "Saddle wBTC/renBTC"
+    const LP_TOKEN_SYMBOL = "saddleEvmosWRenBTC"
+    const INITIAL_A = 400
+    const SWAP_FEE = 4e6 // 4bps
+    const ADMIN_FEE = 0
+
+    await execute(
+      "SwapFlashLoan",
```

Comment:
This will revert since `SwapFlashLoan` instance we are looking at is already initialized. Will need to deploy a new contract instance using `SwapFlashLoan` code